app/src/main/java/org/schabi/newpipe/player/playqueue/PlayQueue.java

```diff
@@ -257,16 +256,6 @@ public synchronized void offsetIndex(final int offset) {
         setIndex(getIndex() + offset);
     }
 
-    /**
-     * Appends the given {@link PlayQueueItem}s to the current play queue.
-     *
-     * @see #append(List items)
-     * @param items {@link PlayQueueItem}s to append
-     */
-    public synchronized void append(@NonNull final PlayQueueItem... items) {
```

Comment:
When this method was called with no arguments it meant that most code in `append` below did nothing except for `broadcast(new AppendEvent(0));`. I would rather create a new method called `notifyChange() { broadcast(new AppendEvent(0)); }` and call it in place of `append(Collections.emptyList()); // Notify change`